apps/core/scenes/results/Results.js

```diff
@@ -130,19 +131,26 @@ class Results extends React.Component<Props> {
     };
     return (
       <SafeAreaView style={styles.container}>
-        <SearchParamsSummary
-          tripType={tripType}
-          departure={{
-            city: travelFromName,
-            localizedDate: getFormattedDate([dateFrom, dateTo]),
-          }}
-          arrival={{
-            city: travelToName,
-            localizedDate: returnDateFrom
-              ? getFormattedDate([returnDateFrom, returnDateTo])
-              : '',
-          }}
-        />
+        {Platform.OS === 'mobile' ? (
```

Comment:
It is still there. I just realised that there is no `mobile` platform. We only have `web`, `ios` and `android`.
So best solution will be to reverse it and use `Platform.OS === 'web'`.

Reply:
@JoseDuSV Done!